api/metric/api.go

```diff
@@ -104,34 +104,60 @@ func (m Measurement) Number() core.Number {
 
 // Meter is an interface to the metrics portion of the OpenTelemetry SDK.
 type Meter interface {
+	MeterMethods
+
+	MeasureConstructors
+	ObserverConstructors
+}
+
+// MeterMethods are all the non-constructor methods of the Meter interface.
+type MeterMethods interface {
```

Comment:
This being one of the interface embedded in multiple others and likely to stick around, can we move away from including `Methods` in the name?

It seems like with the removal for the `LabelSet` this will soon become just a wrapper around `RecordBatch(context.Context, LabelSet, ...Measurement)`. Could this just be called `Batcher`, or maybe `BatchRecorder`?

Reply:
I've moved all the methods back into a single `Meter` interface, and I've removed methods from `MeterMust` so there's no sub-interfaces needed. I figure there's no need for a `BatchRecorder` interface at this time.